tests/drop_counters/mock_server.py

```diff
@@ -0,0 +1,107 @@
+"""
+Utilities for mocking a server using PTF.
+"""
+
+import logging
+import time
+
+from common.platform.device_utils import fanout_switch_port_lookup
+
+class MockServer(object):
```

Comment:
Do you have plans to extend the functionality here? The reason I ask is that the name MockServer is a bit generic. Also, PtfProxy as a name as it appear to funnel/proxy commands to ptf host.

Reply:
I agree. I think this might be good to have in the future, but ArpResponder may suffice for what we're trying to do here. If we do end up keeping this then I'll probably update the name and/or move this to common.